test/visibility/rename/chainInSameUse2.good

```diff
@@ -1,4 +1,5 @@
-chainInSameUse2.chpl:10: warning: identifier 'bleep' is repeated
-note: Did you mean to rename 'a' to 'bloop'?
+chainInSameUse2.chpl:9: In module 'M':
+chainInSameUse2.chpl:10: warning: 'bleep' is repeated
+chainInSameUse2.chpl:10: note: did you mean to rename 'a' to 'bloop'?
```

Comment:
Huh... I'm somewhat surprised that this isn't permitted since it's 1:1 and not taking in or generating any repeats.  But this is obviously no different / worse than what we before (modulo my previous comment about how if the user took the suggested advice, they'd end up still getting a different warning).